agent-install/agent-install.sh

```diff
@@ -1005,6 +1022,25 @@ function urlEquals() {
     fi
 }
 
+# Creates and copies a file similar to os-release on Linux systems to the docker container
+# for anax-in-container installs. This allows the container to know which horizon-cli package
+# to download during automatic upgrades.
+function copy_host_os_info() {
```

Comment:
This function should be moved to horizon-container. This way all the containers, horizon1, horizon2 etc. will have the host-os-release file inside the container.